### ISSUE TITLE
Status

### ISSUE THREAD

**Issue by UnbornHeretic** [blueish4/mod-tools#1]
I can't tell if the tool crashed or is still banning and just taking a long time. A progress amount or a status that it has or has not crashed would be very helpful. That way I can see if I need to break a list into smaller chunks or not.

**Comment by blueish4** [comment]
Hi, thanks for logging this usability issue - I've added an indicator for progress through the list now and also included a projected time to completion in seconds. The completion time is based off of the chat rate limit as set by Twitch, so is only approximate but it'll be fairly close to the real value. 
Out of interest, how many accounts roughly are on the list you're using and are you using the user checker to see if all of them still exist? I guess I should log it as a different issue but there's no effect on Twitch's end for banning a user who doesn't exist and currently the slow banning function doesn't take advantage of the speed of the User API (users can be checked in batches of 100, instead of individually).

I've updated the live version with this change if you want to try it out :)
